.env

```diff
@@ -0,0 +1,5 @@
+ALPINE_RUBY_VERSION=2.6.5
+RAILS_VERSION=5.2.4.1
+SOLR_PORT=8983
+SOLR_URL=http://solr:8983/solr/blacklight-core
+SOLR_VERSION=7
```

Comment:
Is it possible to just run here with `latest` instead? That would be similar to the approach that SolrWrapper used.

Reply:
@mejackreed Yep! I'll make that change.